util/netcdf_generator.py

```diff
@@ -109,7 +109,7 @@ def _setup_coordinate_variables(self, ds):
         # find the depth_variable (10745 AC1)
         depth_variable = find_depth_variable(ds.data_vars)
         coordinates_key = 'coordinates'
-        coordinate_variables = "time lat lon"
+        coordinate_variables = "time lat lon m_gps_lat m_gps_lon"
```

Comment:
I think only the lat and lon (not m_gps_lat and m_gps_lon) will be used as NetCDF file coordinates. If that's true, the original, coordinate_variables = "time lat lon", was correct. It seems bad to have multiple X and Y coordinates in a NetCDF file.

Reply:
I will remove m_gps_lat, m_gps_lon from coordinate_variables. I will also remove them from the "if var == ..." lines below (lines 124, 126).